src/main/fc/cli.c

```diff
@@ -3315,6 +3315,8 @@ static void cliVersion(char *cmdline)
 
 #if defined(USE_RESOURCE_MGMT)
 
+#define MAX_RESOURCE_INDEX(x) (x == 0 ? 1 : x)
```

Comment:
Good pickup. It won't break anything ATM - but I'll raise a PR tomorrow to resolve. Ta.